onadata/libs/utils/csv_import.py

```diff
@@ -305,7 +309,21 @@ def submit_csv(username, xform, csv_file, overwrite=False):
                 if not key.startswith('_') and \
                         (row[key] == 'n/a' or row[key] == ''):
                     del row[key]
-
+                if key in select_multiple_ques:
+                    select_multiple_choices = {
+                        k: v for k, v in row.items() if k.startswith(key)}
+                    for choice_name, choice_value in\
+                            select_multiple_choices.items():
+                        if choice_value == 'True' or choice_value == 1:
+                            split_choice_name = choice_name.split('/')
+                            final_char = split_choice_name[-1]
+                            extra_cols.append(final_char)
+                            result = " ".join(extra_cols)
+                            row[key] = result
+                        elif choice_value not in choice_options:
+                            extra_cols.append(choice_name)
```

Comment:
Not sure about having this `for-loop` within another `for-loop`. If @ukanga thinks this is fine then i'll be fine with it as well